pkg/kubectl/cmd/annotate.go

```diff
@@ -107,7 +109,7 @@ func NewCmdAnnotate(f cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "annotate [--overwrite] (-f FILENAME | TYPE NAME) KEY_1=VAL_1 ... KEY_N=VAL_N [--resource-version=version]",
-		Short:   "Update the annotations on a resource",
+		Short:   T("annotate_short"),
```

Comment:
Still, if none of that sound interesting, one intermediate idea would be

```
T("annotate_short", "Update the annotations on a resource")
```

Eliminates the need of a separate file for the default language which addresses "writing code by having to manually put all your messages in a separate file is painful" and makes the code more readable.